crates/orderbook/openapi.yml

```diff
@@ -579,11 +579,16 @@ components:
         Provides the additional data for ethflow orders
       type: object
       properties:
-        is_refunded:
+        refunded_status:
           description: |
-            Provides a flag that shows whether the refunding service has 
-            refunded the order, in case it was not settled onchain
-          type: boolean
+            Describes the status of the refund
+          type: string
+          enum: 
+            [
+              NonRefundable,
+              NotYetRefunded,
```

Comment:
What will be the value we will get from the orders if it's `NonRefundable`, but we do the manual refund ourselves? If `NonRefundable` is a final state and won't change in that case, I would call the property `automatic_refunded_status` to signal is just the automatic one.


ubernit: I like `NonRefundable` and `Refunded` as final states. However, instead of `NotYetRefunded` I would suggest `PendingAutomaticRefund`. Reason is, it signals that is not refunded but the automatic refund is pending.

Reply:
> PendingAutomaticRefund

But pending suggests that it is already started. But if the order is still open, sending out `PendingAutomaticRefund` will cause confusion, don't you think so?

Reply:
What about `RefundNotStarted` instead of `NotYetRefunded`?

Reply:
for its easier to report only the status when its finished. not intermediate status on whether its already started. This would need more work and can come in a future version

Reply:
I'm still confused if this states are only referring to the "automatic" refund (and not cancelations, etc.)

If so, I would suggest to make it more explicit:
* This PR proposal: `refunded_status`
* My suggestion: `automatic_refund_status`

> for its easier to report only the status when its finished. not intermediate status on whether its already started. This would need more work and can come in a future version

That's fine then. Also, the initiated state can be derived from this enum plus the refundTx (if we do it)